README.md

```diff
@@ -1,31 +1,46 @@
-
 # Apollo Template for ApostropheCMS + Astro integration
 
 This is intended as both a template and starting point for a project with an [ApostropheCMS](https://docs.apostrophecms.org/) backend and frontend powered by [Astro](https://astro.build/).
 
 The ApostropheCMS codebase is located in the `backend` folder of the repository, while the Astro codebase is in the `frontend` folder.
 
 ## Introduction
-Overall, this project utilizes ApostropheCMS as a headless backend with Astro as a frontend. What sets this apart from the typical headless use of ApostropheCMS is the addition of a package, [apostrophe-astro](https://github.com/apostrophecms/apostrophe-astro) in the Astro frontend project. This allows for full use of the ApostropheCMS Admin UI, including in-context editing. At the same time, this package also largely automates and simplifies fetching content from the ApostropheCMS backend without having to make REST API calls.
+Overall, this project utilizes ApostropheCMS as a headless backend with Astro as a frontend. What sets this apart from the typical headless use of ApostropheCMS is the addition of a package, [apostrophe-astro](https://github.com/apostrophecms/apostrophe-astro) in the Astro frontend project. This allows for full use of the ApostropheCMS Admin UI, including in-context editing. At the same time, this package also largely automates and simplifies fetching content from the ApostropheCMS backend without write REST API calls into your code.
 
 ## Using this project
-As outlined briefly above, both this repo and the `apollo-frontend` projects need to be provided with the `APOS_EXTERNAL_FRONT_KEY` environment variable set to the same value. Then, you can start the project as you would any ApostropheCMS project. For example, in a local development environment you can use `npm run dev`. For more details on working with the code in an ApostropheCMS project you can examine our [documentation](https://docs.apostrophecms.org/).
 
-### Similarities to a stand-alone project
-If you have worked with an ApostropheCMS project previously, this repo should look as expected. There are a number of custom modules, providing new pieces, pages, and widgets, located in the `modules` folder. There are also a number of modules that are improving the core Apostrophe modules located in the `modules/@apostrophecms` folder. For a full understanding of Apostrophe you should consult the [documentation](https://docs.apostrophecms.org/), but we will touch on a few highlights later in this document.
+### Prerequisites
+- Node.js v20 or later
+- MongoDB v4.4 or later running on a local server or access to Atlas. See the [ApostropheCMS documentation](https://docs.apostrophecms.org/guide/development-setup.html) for set-up details.
+
+The codebases located in the `backend` and `frontend` folders should be treated as interlinked but separate projects.
+- To start, open a terminal instance at the root of each folder and run `npm install`.
```

Comment:
A top level `install` script is required for our deployment system to work, so you can give them the good news that running `npm install` at the top level is available as a convenience. See `combined-astro-starter-kit`

Reply:
Added info and changed first step.